modules/appeals_api/app/services/appeals_api/events/appeal_received.rb

```diff
@@ -34,28 +27,58 @@ def vanotify_service
         @vanotify_service ||= VaNotify::Service.new(Settings.vanotify.services.lighthouse.api_key)
       end
 
+      def params(template_id)
+        [
+          lookup,
+          template_id,
+          personalisation
+        ].reduce(&:merge)
+      end
+
+      def lookup
+        if opts['email_identifier'] == 'email'
+          {
+            email_address: opts['email_identifier']['id_value']
+          }
+        else
+          {
+            recipient_identifier: {
+              id_value: opts['email_identifier']['id_value'],
+              id_type: opts['email_identifier']['id_type']
+            }
+          }
+        end
+      end
+
       def template_id(template)
         Settings.vanotify.services.lighthouse.template_id.public_send(template)
       end
 
+      def personalisation
+        {
+          'first_name' => opts['first_name'],
+          'date_submitted' => opts['date_submitted'].strftime('%B %d, %Y')
```

Comment:
Probably not worth it yet... But we might want extract this time formatting pattern if we end up using it more...

Reply:
For sure, this was done very quickly to satisfy the AC! I think it can be refactored in the future.